part2/stages/Functions/install-main

```diff
@@ -1033,5 +1033,5 @@ own_tpm() {
 
     generate_policy "${own_key}"
 
-    return 0
+    return $?
```

Comment:
Also, `return $?` is redundant in this context.

Reply:
@jandryuk good point, I don't see any reason to reclaim tpm ownership on an upgrade. In this case I think it would be best to keep the return $?, would you agree? @eric-ch it's possible for generate_policy to return a non-zero value, in which case I would think the user should be alerted and the installation should stop.